docs/Configuration-Settings.md

```diff
@@ -50,26 +52,21 @@ The common configuration contains constants which specify paths and the log appe
 </tr>
 <tr>
   <td>tachyon.logs.dir</td>
-  <td>$tachyon.home + "/logs"</td>
-  <td>Where log files are stored.</td>
+  <td>"${tachyon.home}/logs"</td>
+  <td>The path to store log files.</td>
```

Comment:
I think this doc is outdated, as some configuring options are missing. Also we should sort the options alphabetically.